portfolio/src/main/webapp/script.js

```diff
@@ -71,6 +71,24 @@ function contact() {
 	console.log('Contact!'); 
 }
 
+/** Creates a map and adds it to the page. */
+function createMap() {
+  const map = new google.maps.Map(
+      document.getElementById('map'),
+      {
+          center: {lat: 37.422, lng: -122.084}, 
+          zoom: 16,
+          styles: [
+              {
+               'featureType': 'water',
+               'elementType': 'geometry.fill',
+               'stylers': [{'colors': '#99FF33'}]
```

Comment:
nit - This hex value should be lower case [per style](https://google.github.io/styleguide/htmlcssguide.html#Hexadecimal_Notation).

OOC - can this be `#9f3`, per [style guide](https://google.github.io/styleguide/htmlcssguide.html#Hexadecimal_Notation)?

Reply:
N/A anymore for this PR